lib/headers.sh

```diff
@@ -1,8 +1,10 @@
-# Source all neede libs and helpers, kind of like a main.h.
+# Source all needed libs and helpers, kind of like a main.h.
 
-if [ -z "$TMUX_POWERLINE_DIR_HOME" ]; then
```

Comment:
Bash improvements are appreciated! Though this is not related to fixing #314, thus it would have been best to split this to a separate PR. It would make it easier to review, and also debug if errors are introduced.

Nevertheless, let's continue here. `-z` is not a good idea?

Reply:
ye... I have both `shellharden` and `shellfmt` doing some autoformatting... it felt dirty committing these here, but I did the lazy thing instead of the correct thing :/

Apologies, I should have split these into a separate PR

Turns out that this is actually `shellharden` doing this - using `-z` is technically the correct way to do things, but I've read that comparing against an empty string is useful in terms of readability, particularly when someone's unfamiliar with bash

It seems as though `shellharden` is favouring readability over being pedantic

I'll revert if you prefer - I'm neither here nor there about which style to use :)

Reply:
I see, I'm not familiar with `shellharden`. I'd say let's revert to `-z` for consistency, as it's used everwhere else in this project :)

Reply:
done https://github.com/erikw/tmux-powerline/pull/315/commits/e1f4a7a32c2247227c5e3a9c5b56c6d1105db972 :)